.eslintrc.yml

```diff
@@ -0,0 +1,22 @@
+# AIRBNB code style : https://github.com/airbnb/javascript
+
+env:
+  browser: true
+  es2021: true
+extends:
+  - plugin:react/recommended
+  - airbnb
+overrides: [ ]
+parserOptions:
+  ecmaVersion: latest
```

Comment:
i got a warning from webstorm that latest is not a valid value

Schema validation: Value should be one of: 3, 5, 6, 2015, 7, 2016, 8, 2017, 9, 2018, 10, 2019, 11, 2020, 12, 2021